packages/renderer/src/lib/image/PushImageModal.svelte

```diff
@@ -109,75 +108,81 @@ function callback(name: string, data: string) {
-              ></svg>
-          </button>
-        </div>
-        <!--<form class="px-6 pb-4 space-y-6 lg:px-8 sm:pb-6 xl:pb-8">-->
-        <div class="px-6 pb-4 space-y-6 lg:px-8 sm:pb-6 xl:pb-8">
-          <h3 class="text-xl font-medium text-gray-900 dark:text-white">Pushing image</h3>
-
-          <div>
-            <label for="modalImageTag" class="block mb-2 text-sm font-medium text-gray-900 dark:text-gray-300"
-              >Image Tag</label>
-            <select
-              class="border text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5 bg-gray-600 border-gray-500 placeholder-gray-400 text-white"
-              name="imageChoice"
-              bind:value="{selectedImageTag}">
-              {#each imageTags as imageTag}
-                <option value="{imageTag}">{imageTag}</option>
-              {/each}
-            </select>
-          </div>
-
-          {#if !pushFinished}
+        <div class="relative bg-white rounded-lg shadow dark:bg-gray-700">
+          <div class="flex justify-end p-2">
             <button
-              class="pf-c-button pf-m-primary"
-              disabled="{pushInProgress}"
+              on:click="{() => closeCallback()}"
               type="button"
-              on:click="{() => {
-                pushImage(selectedImageTag);
-              }}">
-              {#if pushInProgress === true}
-                <i class="pf-c-button__progress">
-                  <span class="pf-c-spinner pf-m-md" role="progressbar">
-                    <span class="pf-c-spinner__clipper"></span>
-                    <span class="pf-c-spinner__lead-ball"></span>
-                    <span class="pf-c-spinner__tail-ball"></span>
-                  </span>
-                </i>
-              {:else}
-                <i class="fas fa-arrow-circle-up" aria-hidden="true"></i>
-              {/if}
-              Push image</button>
-          {:else}
-            <button class="pf-c-button pf-m-primary" type="button" on:click="{() => pushImageFinished()}"> Done</button>
-          {/if}
-
-          <div bind:this="{pushLogsXtermDiv}"></div>
-          <!-- </form>-->
+              class="text-gray-400 bg-transparent hover:bg-gray-200 hover:text-gray-900 rounded-lg text-sm p-1.5 ml-auto inline-flex items-center dark:hover:bg-gray-800 dark:hover:text-white"
+              data-modal-toggle="authentication-modal">
+              <svg class="w-5 h-5" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"
+                ><path
+                  fill-rule="evenodd"
+                  d="M4.293 4.293a1 1 0 011.414 0L10 8.586l4.293-4.293a1 1 0 111.414 1.414L11.414 10l4.293 4.293a1 1 0 01-1.414 1.414L10 11.414l-4.293 4.293a1 1 0 01-1.414-1.414L8.586 10 4.293 5.707a1 1 0 010-1.414z"
+                  clip-rule="evenodd"></path
+                ></svg>
+            </button>
+          </div>
+          <!--<form class="px-6 pb-4 space-y-6 lg:px-8 sm:pb-6 xl:pb-8">-->
```

Comment:
I suppose, we can remove this commented code, as far as it redundant now

Reply:
yes, I can add code cleanup on that part

Reply:
done